packages/api/src/services/Service.ts

```diff
@@ -12,30 +12,30 @@ export type Options = {
 };
 
 export default class Service extends EventEmitter {
-  private _client: Client;
+  readonly client: Client;
 
-  private _name: ServiceName;
+  readonly name: ServiceName;
 
-  private _origin: ServiceName;
+  readonly origin: ServiceName;
 
-  private _readyPromise: Promise<null> | undefined;
+  #readyPromise: Promise<null> | undefined;
```

Comment:
I think we can stay with just "private" instead of # in the future. We will have enough typescript checks. 
More details here: https://stackoverflow.com/questions/59641564/what-are-the-differences-between-the-private-keyword-and-private-fields-in-types